src/core/evaluator.js

```diff
@@ -4935,6 +4992,16 @@ class EvalState {
     this._fillColorSpace = this._strokeColorSpace = ColorSpaceUtils.gray;
     this.patternFillColorSpace = null;
     this.patternStrokeColorSpace = null;
+
+    // Path stuff.
+    this.currentPointX = this.currentPointY = 0;
+    this.pathMinMax = new Float32Array([
+      Infinity,
+      Infinity,
+      -Infinity,
+      -Infinity,
+    ]);
+    this.pathBuffer = [];
```

Comment:
*Possibly a stupid question, but:*
Given that objects are passed by reference in JavaScript, do we perhaps want to *copy* these structures when cloning the `EvalState` to avoid affecting a previous save-restore block?

See the `TextState.prototype.clone`-method, and how it handles "objects", for comparison:
https://github.com/mozilla/pdf.js/blob/cd4a30341f048d722e06d078a94911cc19bfc3a6/src/core/evaluator.js#L4921-L4927